pkg/apis/eventing/v1alpha1/broker_validation.go

```diff
@@ -119,20 +126,20 @@ func (b *RabbitBroker) Validate(ctx context.Context) *apis.FieldError {
 	return errs
 }
 
-func ValidateBroker(ctx context.Context, unstructured *unstructured.Unstructured) error {
-	return validate(ctx, unstructured, &RabbitBroker{})
-}
+func (t *RabbitBroker) SetDefaults(ctx context.Context) {}
 
-func validate(ctx context.Context, unstructured *unstructured.Unstructured, t apis.Validatable) error {
-	if unstructured == nil {
+func (b *RabbitBroker) DeepCopyObject() runtime.Object {
```

Comment:
This wasn't autogenerated before?

Reply:
No. But good catch. I was able to generate it by adding another comment.